rdbms/src/main/java/org/spine3/server/storage/jdbc/util/DataSourceWrapper.java

```diff
@@ -30,17 +30,24 @@
 import java.sql.SQLException;
 
 /**
- * The base class for {@link DataSource} wrappers.
+ * Wrapper for {@link DataSource}.
  *
  * @author Alexander Litus
+ * @author Andrey Lavrov
  */
 @Internal
-public abstract class DataSourceWrapper implements AutoCloseable {
+public class DataSourceWrapper implements AutoCloseable {
 
-    /**
-     * Returns the implementation of {@link DataSource}.
-     */
-    public abstract DataSource getDataSource();
+    private final DataSource dataSource;
+
+    /** Wraps custom {@link DataSource} implementation */
```

Comment:
I don't think we have to name all variables with interface type as `custom implementation`, this is rather obvious.